### PR TITLE
Fix Blockchain initialization

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Review by ZhangTao1596 — APPROVED** [review]
Looks good.

**Comment by erikzhang** [comment]
Tested?

File: src/neo/NeoSystem.cs [hunk 1]
```diff
@@ -66,7 +65,7 @@ public NeoSystem(ProtocolSettings settings, string storageEngine = null, string
             this.TaskManager = ActorSystem.ActorOf(Network.P2P.TaskManager.Props(this));
             foreach (var plugin in Plugin.Plugins)
                 plugin.OnSystemLoaded(this);
-            Blockchain.Ask<FillCompleted>(new FillMemoryPool { Transactions = Enumerable.Empty<Transaction>() }).Wait();
+            Blockchain.Ask(GenesisBlock).Wait();
```
Comment (shargon) [inline src/neo/NeoSystem.cs:68]: Use a different message like `BlockchainInit`? we can avoid the condition for all blocks

**Comment by cloud8little** [comment]
Test passed. Error like null exception does not show up anymore.

**Review by cloud8little — APPROVED** [review]

File: src/neo/NeoSystem.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo/NeoSystem.cs]: Done.

**Comment by cloud8little** [comment]
Retest passed.

**Review by cloud8little — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**